build/_k-test.shade

```diff
@@ -38,7 +48,7 @@ projectFile=''
 
         foreach (var framework in targetFrameworks)
         {
-            var testArgs = IsMono ? " -parallel none" : "";
+            var testArgs = (IsMono || singleThreadedTestProjects.Contains(projectName)) ? " -parallel none" : "";
```

Comment:
`Contains` is dangerous here. Use string split and full string match instead. Otherwise you can have a project named `A` for which you want to exclude and it matches project `A.B` too